agents/cdu/fence_cdu.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/python3
```

Comment:
This needs to be `#!@PYTHON@ -tt` to get set based on detected or specified value during configure/build.